spec/features/wrong_word_spec.rb

```diff
@@ -0,0 +1,13 @@
+require "rails_helper"
+
+describe 'test with wrong word' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.